src/Functions/FunctionBinaryArithmetic.h

```diff
@@ -848,11 +848,11 @@ class FunctionBinaryArithmetic : public IFunction
     static FunctionOverloadResolverPtr
     getFunctionForDateTupleOfIntervalsArithmetic(const DataTypePtr & type0, const DataTypePtr & type1, ContextPtr context)
```

Comment:
Should we also later make similar changes to `addTupleOfIntervals`/`subtractTupleOfIntervals`?

Reply:
It would need some changes in the implementation of `addTupleOfIntervals()` / `subtractTupleOfIntervals()`. For consistency, it would be nice to do so. On the other hand, their use case seems very rare. So perhaps this can be a follow-up PR.

Reply:
I meant later in a separate PR, yes :)